File: Python/7kyu/Fizz Buzz.py
# https://www.codewars.com/kata/5300901726d12b80e8000498

def fizzbuzz(n):
    arr = []
    for i in range(1, n+1):
        if i % 3 == 0:
            if i % 5 == 0:
                arr.append("FizzBuzz")
            else:
                arr.append("Fizz")
        elif i % 5 == 0:
            arr.append("Buzz")
        else:
            arr.append(i)
    return arr

File: JavaScript/7kyu/Fizz Buzz.js
// https://www.codewars.com/kata/5300901726d12b80e8000498/

// solution 1
function fizzbuzz(n)
{
    var arr = [];
    for (var i = 1; i <= n; i++) {
        var str = '';
        if (i % 3 == 0) str += 'Fizz';
        if (i % 5 == 0) str += 'Buzz';
        if (str == '') str = i;
        arr.push(str);
    }
    return arr;
}

// solution 2
function fizzbuzz(n)
{
    var arr = [];
    for (var i = 1; i <= n; i++) {
        arr.push(i % 15 == 0 ? 'FizzBuzz' : i % 3 == 0 ? 'Fizz' : i % 5 == 0 ? 'Buzz' : i);
    }
    return arr;
}

console.log(fizzbuzz(19));



